packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -140,7 +140,8 @@ class RequestQueue {
     assert.isType({hash}, 'string');
 
     if(this._queue.includes(hash)) {
-      return await this._idbQDb.get(hash);
+      const req = await this._idbQDb.get(hash);
```

Comment:
@gauntface just wanted to discuss this small change.
actually, this small change fixed everything failing on firefox, do you have any context?